learning/katas/go/core_transforms/combine/lesson-info.yaml

```diff
@@ -18,6 +18,6 @@
 #
 
 content:
-- Simple Function
+- simple_function
 - CombineFn
 - Combine PerKey
```

Comment:
Shouldn't Combine PerKey change to combine_perkey as well?

Reply:
I missed those. Ugh.  👀